docs/help/contents/app-lock.md

```diff
@@ -0,0 +1,43 @@
+# App lock
+
+> error Not available on web
+>
+> App lock is not available on Web.
+
+You can use app lock to restrict access to your app even when your system is unlocked.
+
+## [Desktop](#/tab/desktop)
+
+1. Go to Settings and Click on App lock. Then turn on the App lock switch. You will be prompted to enter your App Lock Password. When it is successful App Lock will be turned on.
+
+<img src="/desktop-enable-app-lock.png" alt="drawing" height="500"/>
+
+2. You can set the time out for your App Lock
+
+<img src="/desktop-lock-app-after.png" alt="drawing" height="500"/>
+
+3. You can also change the pin or password or set a security key.
+
+<img src="/desktop-password-key.png" alt="drawing" height="500"/>
```

Comment:
We should split these into their own headings so its easier to refer a specific App Lock feature to a user. That way we'll also be able to add more information to each topic easily.